kai/data/templates/main.jinja

```diff
@@ -33,21 +33,10 @@ Source file contents:
 
 {% for incident in incidents %}
 ### incident {{ loop.index0 }}
-incident to fix: "{{ incident.analysis_message }}"
-Line number: {{ incident.analysis_line_number }}
-{% if incident.solved_example_diff is defined %}
-Solved example:
-```diff
-{{ incident.solved_example_diff }}
-```{# {% elif incident.solved_example_file is defined %}
-Solved example before changes:
-```{{ incident.src_file_language }}
-{{ incident.solved_example_file.before }}
-```
-Solved example after changes:
-```{{ incident.src_file_language }}
-{{ incident.solved_example_file.after }}
-``` #}
+incident to fix: "{{ incident.message }}"
+Line number: {{ incident.line_number }}
+{% if incident.solution_str is defined %}
+{{ incident.solution_str }}
```

Comment:
Is there still a way for users to override the templates for the solutions?

Reply:
The `get_prompt` still retrieves its prompt from the model provider, with a default of `main.jinja`, so that functionality is still there. Right now, there's no way to override the `solution_handling` templates though.

Reply:
I wonder if we want to just pass variables in from the solution handler to the normal template. Could we declare what variables are declared by a handler and what variables are required for a template so that we can fail on start with an invalid config?

Reply:
I don't think I understand. 

`get_incident_solution_for_file` will pass the same variables to the user-specified prompt regardless. One of those variables, `incident.solution_str`, is generated by `solution_consumer`. Each solution consumer expects a `Solution`, which is indeed type-checked and uses prompts in the `solution_handling` directory. Right now, templates there can't be configured, but the main prompt can.